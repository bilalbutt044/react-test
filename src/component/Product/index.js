import React from 'react';

const Product = ({product}) => {
    return ( 
        <div style={{   height:"200px", width:"250px", border:"1px solid black", paddingLeft:"10px" }}>
            <p> <strong>Name:</strong> {product?.name}</p>
            <p><strong>Description:</strong> {product?.description}</p>
            <p><strong>Price:</strong>{product?.price}</p>
            <p><strong>Date:</strong> {product?.inventoryDate}</p>
        </div>
     );
}
 
export default Product;