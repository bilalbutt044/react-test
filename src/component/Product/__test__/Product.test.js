import renderer from 'react-test-renderer';
import Product from "./../index"

it("render Product component  without error", () => {
    const component = renderer.create(<Product />).toJSON()
    expect(component).toMatchSnapshot() 
})

