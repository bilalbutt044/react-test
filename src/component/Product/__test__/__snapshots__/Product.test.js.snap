// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Product component  without error 1`] = `
<div
  style={
    Object {
      "border": "1px solid black",
      "height": "200px",
      "paddingLeft": "10px",
      "width": "250px",
    }
  }
>
  <p>
     
    <strong>
      Name:
    </strong>
     
  </p>
  <p>
    <strong>
      Description:
    </strong>
     
  </p>
  <p>
    <strong>
      Price:
    </strong>
  </p>
  <p>
    <strong>
      Date:
    </strong>
     
  </p>
</div>
`;
